flutter_local_notifications/example/lib/main.dart

```diff
@@ -1131,6 +1158,19 @@ class _HomePageState extends State<HomePage> {
     return scheduledDate;
   }
 
+  tz.TZDateTime _nextInstanceOfInFiveSecondsLastYear() {
+    final now = tz.TZDateTime.now(tz.local);
```

Comment:
Type needs to be specified. I don't intend to remove this rule. Part of the reason why is Dart allows for dynamics. The 80 line length rule is more of a Flutter/Dart issue. For what it's worth, the Flutter repos requires code to specify types as well.